src/include/session.h

```diff
@@ -222,6 +222,9 @@ struct __wt_session_impl {
     u_int ckpt_handle_next;       /* Next empty slot */
     size_t ckpt_handle_allocated; /* Bytes allocated */
 
+    /* Checkpoint crash for RTS Model testing. */
```

Comment:
This crashpoint logic can be useful for other purposes only, so removing the RTS model and just keeping testing is enough.

Reply:
Fixed in d58ab509ef6a2652ed8bdd2d7578e0f492f06916